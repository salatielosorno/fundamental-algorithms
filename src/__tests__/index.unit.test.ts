import {
  greatestCommonDivisor,
  greatestCommonDivisorOptimized
} from '../index';

describe('Fundamental algorithms tests', () => {
  test('Greatest Common Divisor', () => {
    expect(greatestCommonDivisor({ m: 119, n: 544 })).toMatchSnapshot();
  });

  test('Greatest Common Divisor - Optimized', () => {
    expect(
      greatestCommonDivisorOptimized({ m: 119, n: 544 })
    ).toMatchSnapshot();
  });
});
