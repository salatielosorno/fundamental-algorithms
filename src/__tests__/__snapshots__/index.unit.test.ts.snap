// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fundamental algorithms tests Greatest Common Divisor - Optimized 1`] = `17`;

exports[`Fundamental algorithms tests Greatest Common Divisor 1`] = `17`;
